taniumthreatresponse.json

```diff
@@ -2822,6 +3441,7 @@
         {
             "action": "start quick scan",
             "description": "Scan a computer group for hashes in intel document",
+            "verbose": "This action requires a 'threat response on demand scans write' permission. If this action is failing with 'GroupNotFound' exception then please try with the ID of computer group for 'computer group name' parameter. You can find Computer Group ID from Tanium UI->Administration->Computer Groups->Click on Computer Group name now you can see the ID on URL. If a numerical value is given to the computer group parameter, then priority will be given to 'ID' and ID is not present in tanium, then consider ID as the name of the computer group. If the computer group is deleted but visible on Tanium through the group ID. In this case, this action will be passed using the deleted group ID and scan will start on Tanium without computer group name.",
```

Comment:
Same here

Reply:
Updated verbose as this action require 'threat response on demand scans write' permission.